PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/Invisicam.rbxmx

```diff
@@ -165,6 +164,21 @@ end
 
 -- Update. Called every frame after the camera movement step
 function Invisicam:Update()
+	-- Make sure we still have a Torso
+	if not Torso or not Torso.Parent then
+		local humanoid = Character:FindFirstChild("Humanoid")
```

Comment:
RootCamera already has a fast way to get humanoids, check out the `findPlayerHumanoid()` function there.